extensions/wikia/SpecialDiscussions/SpecialDiscussions.setup.php

```diff
@@ -14,13 +12,14 @@
 ];
 
 // classes
-$wgAutoloadClasses['SpecialDiscussionsController'] = $dir . 'SpecialDiscussionsController.class.php';
+$wgAutoloadClasses['SpecialDiscussionsController'] = __DIR__ . '/SpecialDiscussionsController.class.php';
+$wgAutoloadClasses['SpecialDiscussionsHelper'] = __DIR__ . '/SpecialDiscussionsHelper.php';
 
 // special page
 $wgSpecialPages['Discussions'] = 'SpecialDiscussionsController';
 
 // message files
-$wgExtensionMessagesFiles['SpecialDiscussions'] = $dir . 'SpecialDiscussions.i18n.php';
+$wgExtensionMessagesFiles['SpecialDiscussions'] = __DIR__ . '/SpecialDiscussions.i18n.php';
 
 // permissions
 $wgAvailableRights[] = 'specialdiscussions';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "specialdiscussions" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)